### PR TITLE
Added info about the app

### PR DESCRIPTION
**Pull request by jainendra**

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -1,3 +1,6 @@
 # Pandit.AI
 - Dictates pooja steps
     - It waits for the feedback before dictating next step
+- It will use Hindi language to dicatate pooja steps
+- It will listen for words such as "ho gaya" to move to next step of pooja
+- It will also have ability to play bhakti geet
```
Comment (dewa-himanshu24) [inline README.MD:6]: Please use "song" instead of  "geet".

Comment (jainendra) [reply README.MD:6]: I think this still looks good. I would say lets stick with geet only

Comment (dewa-himanshu24) [reply README.MD:6]: okay

**Comment by dewa-himanshu24** [comment]
Approved

**Pull request merged**